lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,85 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+const feedbackMessages = {
```

Comment:
> But I have no preferences on the logic / mechanisms used to achieve that.

Although having said that, I guess it depends if we feel the unsafe-mode setting is part of the core application itself, or whether it's "just" a concern of the GUI sitting on top? (IYSWIM!)